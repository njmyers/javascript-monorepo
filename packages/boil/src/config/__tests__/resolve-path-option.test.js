import resolvePathOption from "../resolve-path-option";

describe("it works", () => {
  test("it is a function ", () => {
    expect(typeof resolvePathOption()).toBe("function");
  });

  test("it runs without crashing", () => {
    expect(resolvePathOption()).toMatchSnapshot();
  });
});
