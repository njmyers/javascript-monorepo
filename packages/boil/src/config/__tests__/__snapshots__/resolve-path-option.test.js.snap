// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it works it runs without crashing 1`] = `""`;
